_includes/api/en/4x/req-is.md

```diff
@@ -9,13 +9,13 @@ Returns `false` otherwise.
 req.is('html');
 req.is('text/html');
 req.is('text/*');
-// => true
+// => 'html'
 
 // When Content-Type is application/json
 req.is('json');
 req.is('application/json');
 req.is('application/*');
-// => true
+// => 'html'
```

Comment:
Of course, none of these will return the string `'html'`, but rather they will all return their own input string.

Reply:
🤦‍♂️ yea that was a fat-finger mistake ...